### ISSUE TITLE
Metadata URLs

### ISSUE THREAD

**Issue by ghobona** [opengeospatial/ogcapi-features#477]
Please see https://github.com/opengeospatial/oapi_common/issues/51

**Comment by cportele** [comment]
@ghobona - Was this intended to be moved to the Common repository?

**Comment by ghobona** [comment]
@cportele You are right. It was intended for OGC API - Common. I will move it there.
